devito/symbolics/printer.py

```diff
@@ -21,7 +21,8 @@ class CodePrinter(C99CodePrinter):
         'FLOATP': '(float*)',
         'DOUBLE': '(double)',
         'DOUBLEP': '(double*)',
-    }
+        'Min': 'MIN',
```

Comment:
probably unnecessary. 

`relax_incr_dimensions` _defines_ the min/max names, so it can use MIN/MAX directly instead of min/max (eg via `Function('MIN')`), isn't this enough?

Reply:
Need to add those in extended sympy though?
```
MIN = Function('MIN')
MAX = Function('MAX')

```